### ISSUE TITLE
[Bug]: MudExDialogService does not implement new 

### ISSUE THREAD

**Issue by jaypalexa** [fgilde/MudBlazor.Extensions#98]
### What happened?

Florian -- I know you are on vacation until August 20th, and I do not expect a resolution before then, of course.  I just wanted to get this documented before I forgot about it.

---

The following exception is thrown on startup in `Program.cs` when executing `builder.Services.AddMudExtensions();`:

```
System.Reflection.ReflectionTypeLoadException: 'Unable to load one or more of the requested types.
Method 'add_DialogInstanceAddedAsync' in type 'MudBlazor.Extensions.Services.MudExDialogService' from assembly 'MudBlazor.Extensions, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.'
```

![image](https://github.com/user-attachments/assets/e864dd67-f0c4-495e-bf47-51844c44bf82)

---

It looks as though MudBlazor v.7.2.0 possibly introduced a breaking change by adding a new `DialogInstanceAddedAsync` event to the `IDialogService` interface:

![image](https://github.com/user-attachments/assets/adda526d-f816-4be1-945d-45834a5e464a)

---
refs:
https://github.com/MudBlazor/MudBlazor/pull/9306

### Expected Behavior

No exception on startup

### What application type are you referring to?

WebAssembly

### MudBlazor.Extension Version

2.0.0

### MudBlazor Version

7.2.0

### What .net Version are you using?

.Net8

### What browser are you using?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fgilde** [comment]
Thanks for reporting. And I'm really sorry that my vacation is until start of mud 7. all open bugs are very easy to fix but currently I can't do it. Maybe next week we have a stop with a better house and internet, maybe I can fix it then.
However thank you for reporting.

**Comment by ScarletKuro** [comment]
I'm really sorry for this breaking change. I should have provided a default implementation since C# allows it for interfaces now for exact this reason.

**Comment by jaypalexa** [comment]
> I'm really sorry for this breaking change. I should have provided a default implementation since C# now allows it for interfaces for this exact reason.

@ScarletKuro Oooh...that's an interesting and good idea.  Any chance of tweaking that for the next MudBlazor release?

**Comment by fgilde** [comment]
So I used the mac book from my wife and created a fix.

Its fixed in 2.0.2 https://www.nuget.org/packages/MudBlazor.Extensions/2.0.2

**Comment by jaypalexa** [comment]
All working for me here now, many thanks.  You have gone above and beyond.  Enjoy your vacation!
